Code/Engine/Foundation/Reflection/Implementation/ArrayProperty.h

```diff
@@ -207,3 +207,56 @@ class ezMemberArrayProperty : public ezTypedArrayProperty<typename ezTypeTraits<
   GetConstContainerFunc m_ConstGetter;
   GetContainerFunc m_Getter;
 };
+
+template <typename Class, typename Container, typename Type>
+class ezMemberArrayReadOnlyProperty : public ezTypedArrayProperty<typename ezTypeTraits<Type>::NonConstReferenceType>
```

Comment:
@Sanakan8472 is this property needed?

Also, if yes, it should have some code comment describing what effect it has.

Reply:
It is needed to reflect ezArrayPtr as member